src/test/java/com/synopsys/integration/jenkins/scan/service/bridge/BridgeDownloadParameterServiceTest.java

```diff
@@ -88,6 +88,53 @@ void getBridgeDownloadParamsTest() {
         assertEquals("/path/to/bridge", result.getBridgeInstallationPath());
     }
 
+    @Test
+    void getBridgeDownloadParamsTestWithAirgapEnabledAndVersion() {
```

Comment:
The method name should be `getBridgeDownloadParamsWithAirgapEnabledAndVersionTest()` so that all test method names are consistent.

Reply:
Done.